docs/reference/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -52,11 +52,200 @@ For more information about versions, build numbers, and Office Online Server, se
+||[getItemByReplyId(replyId: string)](/javascript/api/excel/excel.commentcollection#getitembyreplyid-replyid-)|Gets a comment related to its reply ID in the collection.|
+||[load(option?: OfficeExtension.LoadOption)](/javascript/api/excel/excel.commentcollection#load-option-)|Queues up a command to load the specified properties of the object. You must call "context.sync()" before reading the properties.|
+||[items](/javascript/api/excel/excel.commentcollection#items)|Gets the loaded child items in this collection.|
+|[CommentReply](/javascript/api/excel/excel.commentreply)|[content](/javascript/api/excel/excel.commentreply#content)|Get or set the content.|
+||[delete()](/javascript/api/excel/excel.commentreply#delete--)|Deletes the comment reply.|
+||[getLocation()](/javascript/api/excel/excel.commentreply#getlocation--)|Get location of the comment reply.|
+||[getParentComment()](/javascript/api/excel/excel.commentreply#getparentcomment--)|Get its parent comment of this reply.|
+||[authorEmail](/javascript/api/excel/excel.commentreply#authoremail)|Get author email of the comment reply.|
+||[authorName](/javascript/api/excel/excel.commentreply#authorname)|Get author name of the comment reply.|
+||[creationDate](/javascript/api/excel/excel.commentreply#creationdate)|Get creation time of the comment reply.|
+||[id](/javascript/api/excel/excel.commentreply#id)|Represents the comment reply identifier. Read-only.|
+|[CommentReplyCollection](/javascript/api/excel/excel.commentreplycollection)|[add(content: string, contentType?: "Plain")](/javascript/api/excel/excel.commentreplycollection#add-content--contenttype-)|Creates a comment reply for comment.|
+||[add(content: string, contentType?: Excel.ContentType)](/javascript/api/excel/excel.commentreplycollection#add-content--contenttype-)|Creates a comment reply for comment.|
+||[getCount()](/javascript/api/excel/excel.commentreplycollection#getcount--)|Gets the number of comment replies in the collection.|
+||[getItem(commentReplyId: string)](/javascript/api/excel/excel.commentreplycollection#getitem-commentreplyid-)|Returns a comment reply identified by its ID. Read-only.|
+||[getItemAt(index: number)](/javascript/api/excel/excel.commentreplycollection#getitemat-index-)|Gets a comment reply based on its position in the collection.|
+||[load(option?: OfficeExtension.LoadOption)](/javascript/api/excel/excel.commentreplycollection#load-option-)|Queues up a command to load the specified properties of the object. You must call "context.sync()" before reading the properties.|
+||[items](/javascript/api/excel/excel.commentreplycollection#items)|Gets the loaded child items in this collection.|
+|[CustomFunctionEventArgs](/javascript/api/excel/excel.customfunctioneventargs)|[higherTicks](/javascript/api/excel/excel.customfunctioneventargs#higherticks)||
```

Comment:
missing description?

Reply:
I'll tag @mscharlock on this, Here's the [reference doc page](https://docs.microsoft.com/en-us/javascript/api/excel/excel.customfunctioneventargs?view=office-js).